simpledotorg/simple-server
# Remove drug stock calculation from progress tab

**Story card:** [sc-10472](https://app.shortcut.com/simpledotorg/story/10472/drug-stock-calculation-is-being-displayed-at-the-bottom-of-the-progress-tab)

## Because

Drug stock calculation is displayed at the bottom of the Progress Tab after a drug stock report is submitted

## This addresses

- Removes the unnecessary drug stock calculation from the progress tab
- Removes some unused files related to drug stock calculation

## Test instructions

- Checkout to the current branch 
- Go to `http://localhost:3000/api/v3/analytics/user_analytics.html`
  - Get the user credentials using `rails get_user_credentials` from the terminal
  - Add the credentials to the request headers
 
  The calculation appears only after you submit a drug stock report for a month
- Now, submit the drug stock report for the month and click done
- Now go back to the main page on the progress tab and check if the drug stock calculation is displayed or not
- All good if the drug stock calculation is not displayed on the bottom of the progress tab

Screenshots:
After:
<img width="250" alt="Screenshot 2023-04-28 at 12 24 37 PM" src="https://user-images.githubusercontent.com/32141642/235076367-3ed12e67-e7fe-49b7-8550-c2716e88b9d4.png">

Before:
<img width="250" alt="Screenshot 2023-04-28 at 12 24 59 PM" src="https://user-images.githubusercontent.com/32141642/235076379-5b7c1ef4-3404-441c-85ed-00e361077d65.png">
